samples/webhook.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START dialogflow_es_webhook]
+
+exports.handleWebhook = (request, response) => {
+  const tag = request.body.queryResult.intent.displayName;
+  console.log('HERE');
```

Comment:
Did you want these `console.logs` left in?

Reply:
No that was for debugging i'll remove it